src/feeds.js

```diff
@@ -14,6 +14,9 @@ export default class Feeds {
     logger,
   } = {}) {
     this.authData = authData;
+    if (!authEndpoint) {
+      console.warn("No authEndpoint provided; subscriptions to private feeds will fail.");
+    }
```

Comment:
I don't think we should have this. It's perfectly valid to only use public feeds and not require an auth endpoint.

I saw you mentioned this @zmarkan. Maybe I'm missing something?